example.yml

```diff
@@ -1,6 +1,7 @@
 # ISA Configurations
 isa_params:  
   xlen : 32
+  pmp_napot_gran : 14
```

Comment:
This is sort of hard to understand from a user perspective. Could we do something like "pmp_minimum_granularity" and have human-readable byte values, e.g. `4, 8, 16, 32, 64, 128, 256, 512, 1K, 2K, 4K, ...`, then calculate the "G" value in the config_core script?

@devins2518 See if you agree.

Reply:
Yeah I was also thinking that. I think 4bytes-16k would be a reasonable range. The python script could probably do the G conversion easier than in systemverilog (a dictionary should do the job fine I think)

Reply:
Just updated this in the yml & sv files. What should I do about utility.h?

Reply:
Ideally we'd have some sort of utility to produce header files with defines congruent to the current build that can plug and play everywhere, but since we don't I think its okay to leave it as is. If you want, you can leave a comment with the dictionary based mapping just so its a bit easier to find.